README.md

```diff
@@ -1,59 +1,23 @@
 # Joomla Cypress
 This is a support package that helps in writing end-to-end tests for the [Joomla CMS](https://joomla.org) and its extensions with the frontend testing tool [Cypress](/https://www.cypress.io/).
 
+Joomla versions 4 and 5, as well as all active Joomla development branches, are supported.
```

Comment:
I would remove 4 and 5 and just mention the active development branches.

Reply:
Removed naming Joomla version